commit/core.pyx

```diff
@@ -94,6 +95,7 @@ cdef class Evaluation :
         self.THREADS    = None # set by "set_threads" method
         self.A          = None # set by "build_operator" method
         self.x          = None # set by "fit" method
+        self.confidence_map_img = None # set by "fit" method
```

Comment:
Maintain the indentation (looks nicer)

Reply:
Yes, it is nice and I like it. 
About the style to write code, this is a proposal https://realpython.com/python-pep8/ , we can use some of their recomendations.